events/cloudwatch_events.go

```diff
@@ -5,9 +5,9 @@ import (
 	"time"
 )
 
-// CloudWatchEvent is the outer structure of an event sent via CloudWatch Events.
-// For examples of events that come via CloudWatch Events, see https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html
-type CloudWatchEvent struct {
```

Comment:
Renaming the type is a breaking change. Adding a type alias `type EventBridgeEvent = CloudWatchEvent` would be required to not break existing consumers